app/jobs/consume_assessments_job.rb

```diff
@@ -146,16 +146,16 @@ def perform
         else
           # If message['reported_symptoms_array'] is not populated then this assessment came in through
           # a generic channel ie: SMS where monitorees are asked YES/NO if they are experiencing symptoms
+          typed_reported_symptoms = if message['experiencing_symptoms']
```

Comment:
Actually, isn't this problematic because it won't create new Symptom records for each dependent anymore?

Reply:
Yes